webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9158,6 +9187,31 @@ var userOverride = {
                                 : {}
                         );
                     }
+                } else if (
+                    !(
+                        msg.metadata.obsolete && msg.metadata.obsolete == 'true'
+                    ) &&
+                    msg.metadata.KM_FIELD
+                ) {
+                    var fieldMetadata = {};
+                    try {
+                        fieldMetadata = JSON.parse(msg.metadata.KM_FIELD);
+                    } catch (e) {
+                         console.error('fieldMetadata should not be empty');
+                    }
+                    var fieldValidation = fieldMetadata.validation.regex;
+                    var errorMessage = fieldMetadata.validation.errorText;
+                    $mck_text_box
+                        .addClass('mck-text-box')
+                        .removeClass('n-vis')
+                        .data('validation', fieldValidation);
+                    $mck_text_box.data('ErrorMessage', errorMessage);
```

Comment:
'ErrorMessage' i guess this is a custom data- attribute?

if yes, we should follow the naming conventions